pkg/saga/statemachine/statelang/parser/statemachine_parser.go

```diff
@@ -52,22 +41,118 @@ func (b BaseStateParser) GetString(stateName string, stateMap map[string]interfa
 	return valueAsString, nil
 }
 
-func (b BaseStateParser) GetSlice(stateName string, stateMap map[string]interface{}, key string) ([]interface{}, error) {
+func (b BaseStateParser) GetStringOrDefault(stateMap map[string]interface{}, key string, defaultValue string) string {
 	value := stateMap[key]
+	if value == nil {
+		return defaultValue
+	}
+
+	valueAsString, ok := value.(string)
+	if !ok {
```

Comment:
if value convert err, example need: int , but map , throw err or better

Reply:
I throw err in all methods now.